### PR TITLE
Removed cargo delivery dependency

### PR DESCRIPTION
**Pull request by pijaro**

### CODE REVIEW

File: mock/cpp_mock_scenarios/src/collision/crashing_npc.cpp [hunk 1]
```diff
@@ -30,10 +29,7 @@ class IdiotNpcScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit IdiotNpcScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "crashing_npc",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/collision/crashing_npc.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/collision/spawn_with_offset.cpp [hunk 2]
```diff
@@ -30,10 +29,7 @@ class SpawnWithOffsetScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit SpawnWithOffsetScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "spawn_with_offset",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/collision/spawn_with_offset.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/cpp_scenario_node.cpp [hunk 3]
```diff
@@ -18,11 +18,9 @@
 namespace cpp_mock_scenarios
 {
 CppScenarioNode::CppScenarioNode(
-  const std::string & node_name, const std::string & map_path,
-  const std::string & lanelet2_map_file, const std::string & scenario_filename, const bool verbose,
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/cpp_scenario_node.cpp:22]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/crosswalk/stop_at_crosswalk.cpp [hunk 4]
```diff
@@ -30,9 +29,7 @@ class StopAtCrosswalkScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit StopAtCrosswalkScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "stop_at_crosswalk", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/crosswalk/stop_at_crosswalk.cpp:34]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/follow_front_entity/accelerate_and_follow.cpp [hunk 5]
```diff
@@ -30,10 +29,7 @@ class AccelerateAndFollowScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit AccelerateAndFollowScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "accelerate_and_follow",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/follow_front_entity/accelerate_and_follow.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/follow_front_entity/decelerate_and_follow.cpp [hunk 6]
```diff
@@ -30,10 +29,7 @@ class DecelerateAndFollowScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit DecelerateAndFollowScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "decelerate_and_follow",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/follow_front_entity/decelerate_and_follow.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/follow_lane/acquire_position_in_world_frame.cpp [hunk 7]
```diff
@@ -30,9 +29,7 @@ class DecelerateAndFollowScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit DecelerateAndFollowScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "idiot_npc", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/follow_lane/acquire_position_in_world_frame.cpp:34]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/lane_change/lanechange_left.cpp [hunk 8]
```diff
@@ -30,9 +29,7 @@ class LaneChangeLeftScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit LaneChangeLeftScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "lanechange_left", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/lane_change/lanechange_left.cpp:34]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/measurement/get_longitudinal_distance.cpp [hunk 9]
```diff
@@ -30,10 +29,7 @@ class GetLongitudinalDistanceScenario : public cpp_mock_scenarios::CppScenarioNo
 {
 public:
   explicit GetLongitudinalDistanceScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "get_longitudinal_distance",
-      ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map", "lanelet2_map.osm",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/measurement/get_longitudinal_distance.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/merge/merge_left.cpp [hunk 10]
```diff
@@ -30,9 +29,7 @@ class MergeLeftScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MergeLeftScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "merge_left", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/merge/merge_left.cpp:34]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/metrics/momentary_stop.cpp [hunk 11]
```diff
@@ -29,10 +28,7 @@ class MomentaryStopScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MomentaryStopScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "momentary_stop",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/metrics/momentary_stop.cpp:34]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/metrics/out_of_range.cpp [hunk 12]
```diff
@@ -30,10 +29,7 @@ class OutOfRangeScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit OutOfRangeScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "out_of_range",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/metrics/out_of_range.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/metrics/reaction_time.cpp [hunk 13]
```diff
@@ -30,10 +29,7 @@ class ReactionTimeScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit ReactionTimeScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "reaction_time",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/metrics/reaction_time.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/metrics/traveled_distance.cpp [hunk 14]
```diff
@@ -30,10 +29,7 @@ class TraveledDistanceScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit TraveledDistanceScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "traveled_distance",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/metrics/traveled_distance.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/move_backward/move_backward.cpp [hunk 15]
```diff
@@ -30,10 +29,7 @@ class MoveBackwardScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MoveBackwardScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "move_backward",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/move_backward/move_backward.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/traffic_light/phase_control.cpp [hunk 16]
```diff
@@ -30,10 +29,7 @@ class PhaseControlScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit PhaseControlScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "phase_control",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/traffic_light/phase_control.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: mock/cpp_mock_scenarios/src/traffic_simulation_demo.cpp [hunk 17]
```diff
@@ -30,10 +29,7 @@ class TrafficSimulationDemoScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit TrafficSimulationDemoScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "traffic_simulation_demo",
-      ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map", "lanelet2_map.osm",
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/src/traffic_simulation_demo.cpp:35]: .osm file should be exactly fixed in scenario.
So, please specify it in constructor.

File: .github/workflows/ScenarioTest.yaml [hunk 18]
```diff
@@ -91,104 +91,104 @@ jobs:
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=crashing_npc timeout:=5 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=crashing_npc osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=5 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:94]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 19]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run spawn with offset scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=spawn_with_offset timeout:=15
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=spawn_with_offset osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:101]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 20]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run traffic simulation demo scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traffic_simulation_demo timeout:=70 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traffic_simulation_demo map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=70 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:108]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 21]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run accelerate and follow front entity scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=accelerate_and_follow timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=accelerate_and_follow osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:115]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 22]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run decelerate_and_follow front entity scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=decelerate_and_follow timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=decelerate_and_follow osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:122]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 23]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run lanechange_left scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=lanechange_left timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=lanechange_left map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:129]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 24]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run merge_left scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=merge_left timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=merge_left map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:136]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 25]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run phase_control scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=phase_control timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=phase_control osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:143]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 26]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run traveled_distance scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traveled_distance timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=traveled_distance osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:150]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 27]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run momentary stop scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=momentary_stop timeout:=20 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=momentary_stop osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=20 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:157]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 28]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run reaction_time scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time timeout:=10 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=10 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:164]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 29]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run out_of_range scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=out_of_range timeout:=7 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=out_of_range osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=7 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:171]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 30]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run stop_at_crosswalk scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=stop_at_crosswalk timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=stop_at_crosswalk map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:178]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 31]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run move_backward scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=move_backward timeout:=10 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=move_backward osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=10 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:185]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: .github/workflows/ScenarioTest.yaml [hunk 32]
```diff
@@ -91,104 +91,104 @@ jobs:
...
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run get_longitudinal_distance scenario
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=get_longitudinal_distance timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=get_longitudinal_distance map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```
Comment (hakuturu583) [inline .github/workflows/ScenarioTest.yaml:191]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: mock/cpp_mock_scenarios/launch/mock_test.launch.py [hunk 33]
```diff
@@ -75,11 +75,17 @@ def generate_launch_description():
...
     scenario = LaunchConfiguration("scenario", default="")
     launch_rviz = LaunchConfiguration("launch_rviz", default=False)
+    map_path = LaunchConfiguration("map_path", default=str(Path.cwd()))
+    osm_file = LaunchConfiguration("osm_file", default="lanelet2_map.osm")
     scenario_node = Node(
         package="cpp_mock_scenarios",
         executable=scenario,
         name=scenario,
         output="screen",
+        parameters=[{
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/launch/mock_test.launch.py:85]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

File: mock/cpp_mock_scenarios/launch/mock_test.launch.py [hunk 34]
```diff
@@ -75,11 +75,17 @@ def generate_launch_description():
     timeout = LaunchConfiguration("timeout", default=10)
     scenario = LaunchConfiguration("scenario", default="")
     launch_rviz = LaunchConfiguration("launch_rviz", default=False)
+    map_path = LaunchConfiguration("map_path", default=str(Path.cwd()))
```
Comment (hakuturu583) [inline mock/cpp_mock_scenarios/launch/mock_test.launch.py:78]: .osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.

**Review by hakuturu583 — CHANGES_REQUESTED** [review]

File: mock/cpp_mock_scenarios/src/collision/crashing_npc.cpp (hunk 1 above)
Comment (pijaro) [reply mock/cpp_mock_scenarios/src/collision/crashing_npc.cpp:35]: Constructor has map path and osm file parameters again.

File: .github/workflows/ScenarioTest.yaml (hunk 30 above)
Comment (pijaro) [reply .github/workflows/ScenarioTest.yaml:178]: Reverted cpp_mock_scenarios parameters. It has no parameters now.

**Review by hakuturu583 — APPROVED** [review]
LGTM!

**Pull request merged**